SparkPost/2web2ui
# FE-1246 - Create Empty state for A/B Testing page and Add banner to A/B testing page

FE-1246 - Create Empty state for A/B Testing page and Add banner to A/B testing page

### What Changed

- Added Empty state for A/B Testing page - https://sparkpost.invisionapp.com/console/Empty-States-ckcno05yk00tf011ij29ebpmx/ckevlcnj906cu01zkbus96r8h/play#project_console
- Added banner to A/B testing Listpage - https://sparkpost.invisionapp.com/console/Empty-States-ckcno05yk00tf011ij29ebpmx/ckevlcnja06cv01zk7onya4eg/play#project_console
- Send an event to Segment when the user sees empty state 

### How To Test
 - Enable account ui flag - "allow_empty_states"
 - When A/B tests = 0 and hibana is enabled, verify you see the Empty state.
 - verify that a event is sent to segment when empty state loads
 - When A/B tests > 0 and hibana is enabled, verify you see the Banner.
      - Verify once the banner is dismissed it doesn't show up again

### To Do
